src/content/docs/en/reference/api-reference.mdx

```diff
@@ -807,29 +924,40 @@ The `CollectionEntry<TCollectionName>` type is an object with the following valu
 
 #### `id`
 
-**Example Type:** `'entry-1.md' | 'entry-2.md' | ...`
+**Example Type for "content" collections:** `'entry-1.md' | 'entry-2.md' | ...`  
+**Example Type for "data" collections:** `'author-1' | 'author-2' | ...`
 
-A unique ID using the file path relative to `src/content/[collection]`. Enumerates all possible string values based on the collection entry file paths.
+A unique ID using the file path relative to `src/content/[collection]`. Enumerates all possible string values based on the collection entry file paths. Note that collections [defined as `type: 'content'`](#type) include the file extension in their ID, while collections defined as `type: 'data'` do not.
 
-#### `slug`
+#### `collection`
 
-**Example Type:** `'entry-1' | 'entry-2' | ...`
+**Example Type:** `'blog' | 'authors' | ...`
 
-A URL-ready slug. Defaults to the `id` without the file extension, but can be overridden by setting [the `slug` property](/en/guides/content-collections/#defining-custom-slugs) in a file's frontmatter.
+The name of the entry's collection.
```

Comment:
```suggestion
The name of a top-level folder under `src/content/` in which entries are located. This is the name used to reference the collection in your schema, and in querying functions.
```
I think we do need to connect this to the directory/folder name, as well as mention that this is what we consider to be the collection's "name" and that it's used in the schema, functions etc. I just blurted this suggestion above out quickly, but that's kind of the idea I'm thinking.